OpenRA.Game/Network/UnitOrders.cs

```diff
@@ -45,7 +45,7 @@ public static void ProcessOrder(OrderManager orderManager, World world, int clie
 							var suffix = (player != null && player.WinState == WinState.Lost) ? " (Dead)" : "";
 							suffix = client.IsObserver ? " (Spectator)" : suffix;
 
-							if (orderManager.LocalClient != null && client != orderManager.LocalClient && client.Team == orderManager.LocalClient.Team)
+							if (orderManager.LocalClient != null && client != orderManager.LocalClient && client.Team > 0 && client.Team == orderManager.LocalClient.Team)
```

Comment:
Possibly worth mentioning that because this is based on client team IDs, it will interact badly with anyone doing fragile alliances. (which are a bad idea anyway and should be permanently nuked... )